docs/api-types/issuer/spec.md

```diff
@@ -142,6 +142,22 @@ cloudflare:
     key: api-key
 ```
 
+##### Akamai FastDNS
+
+```yaml
+akamai:
+  serviceConsumerDomain: akab-tho6xie2aiteip8p-poith5aej0ughaba.luna.akamaiapis.net
```

Comment:
Just to check: one service consumer domain can be used to manage multiple DNS zones, right? Or does this field dictate which DNS zone can be updated?

Reply:
Yes, those service consumer domains are connected to their respective Akamai account. The domain can therefore be used to manage all zones belonging to that account. AFAIK there's no way to restrict the zones.